coverconfig.json

```diff
@@ -0,0 +1,16 @@
+{
+    "enabled": true,
+    "relativeSourcePath": "../../../out",
```

Comment:
here is the difference with vscod-tekton configuration. They are pointing to src but we are generating js file sin out folder.